doc/api/fs.md

```diff
@@ -7404,7 +7404,101 @@ added:
+| HPFS\_SUPER\_MAGIC      | `0xf995e849` | `4187351113` |
+| HUGETLBFS\_MAGIC        | `0x958458f6` | `2508478710` |
+| ISOFS\_SUPER\_MAGIC     | `0x9660`     | `38496`      |
+| JFFS2\_SUPER\_MAGIC     | `0x72b6`     | `29366`      |
+| JFS\_SUPER\_MAGIC       | `0x3153464a` | `827541066`  |
+| MINIX\_SUPER\_MAGIC     | `0x137f`     | `4991`       |
+| MINIX\_SUPER\_MAGIC2    | `0x138f`     | `5007`       |
+| MINIX2\_SUPER\_MAGIC    | `0x2468`     | `9320`       |
+| MINIX2\_SUPER\_MAGIC2   | `0x2478`     | `9336`       |
+| MINIX3\_SUPER\_MAGIC    | `0x4d5a`     | `19802`      |
+| MQUEUE\_MAGIC           | `0x19800202` | `427819522`  |
+| MSDOS\_SUPER\_MAGIC     | `0x4d44`     | `19780`      |
+| MTD\_INODE\_FS\_MAGIC   | `0x11307854` | `288389204`  |
+| NCP\_SUPER\_MAGIC       | `0x564c`     | `22092`      |
+| NFS\_SUPER\_MAGIC       | `0x6969`     | `26985`      |
+| NILFS\_SUPER\_MAGIC     | `0x3434`     | `13364`      |
+| NSFS\_MAGIC             | `0x6e736673` | `1853056627` |
+| NTFS\_SB\_MAGIC         | `0x5346544e` | `1397118030` |
+| OCFS2\_SUPER\_MAGIC     | `0x7461636f` | `1952539503` |
+| OPENPROM\_SUPER\_MAGIC  | `0x9fa1`     | `40865`      |
+| OVERLAYFS\_SUPER\_MAGIC | `0x794c7630` | `2035054128` |
+| PIPEFS\_MAGIC           | `0x50495045` | `1346981957` |
+| PROC\_SUPER\_MAGIC      | `0x9fa0`     | `40864`      |
+| PSTOREFS\_MAGIC         | `0x6165676c` | `1634035564` |
+| QNX4\_SUPER\_MAGIC      | `0x002f`     | `47`         |
+| QNX6\_SUPER\_MAGIC      | `0x68191122` | `1746473250` |
+| RAMFS\_MAGIC            | `0x858458f6` | `2240043254` |
+| REISERFS\_SUPER\_MAGIC  | `0x52654973` | `1382369651` |
+| ROMFS\_MAGIC            | `0x7275`     | `29301`      |
+| SECURITYFS\_MAGIC       | `0x73636673` | `1935894131` |
+| SELINUX\_MAGIC          | `0xf97cff8c` | `4185718668` |
+| SMACK\_MAGIC            | `0x43415d53` | `1128357203` |
+| SMB\_SUPER\_MAGIC       | `0x517b`     | `20859`      |
+| SMB2\_MAGIC\_NUMBER     | `0xfe534d42` | `4266872130` |
+| SOCKFS\_MAGIC           | `0x534f434b` | `1397703499` |
+| SQUASHFS\_MAGIC         | `0x73717368` | `1936814952` |
+| SYSFS\_MAGIC            | `0x62656572` | `1650812274` |
+| SYSV2\_SUPER\_MAGIC     | `0x012ff7b6` | `19920822`   |
+| SYSV4\_SUPER\_MAGIC     | `0x012ff7b5` | `19920821`   |
+| TMPFS\_MAGIC            | `0x01021994` | `16914836`   |
+| TRACEFS\_MAGIC          | `0x74726163` | `1953653091` |
+| UDF\_SUPER\_MAGIC       | `0x15013346` | `352400198`  |
+| UFS\_MAGIC              | `0x00011954` | `72020`      |
+| USBDEVICE\_SUPER\_MAGIC | `0x9fa2`     | `40866`      |
+| V9FS\_MAGIC             | `0x01021997` | `16914839`   |
+| VXFS\_SUPER\_MAGIC      | `0xa501fcf5` | `2768370933` |
+| XENFS\_SUPER\_MAGIC     | `0xabba1974` | `2881100148` |
+| XENIX\_SUPER\_MAGIC     | `0x012ff7b4` | `19920820`   |
+| XFS\_SUPER\_MAGIC       | `0x58465342` | `1481003842` |
+| \_XIAFS\_SUPER\_MAGIC   | `0x012fd16d` | `19911021`   |
+
+</details>
```

Comment:
Thank you for the comments. 
If you run `fs.statfs(path)`, it returns a decimal and not a hex number. I wanted to add it in so that if someone searched for the decimal number, it would direct them here.
I'll add in your recommendation with a note regarding the output.

Reply:
Strictly speaking, it returns just a number (agnostic to any numerical system). It only becomes decimal or hexadecimal when stringified.
Nevertheless, I agree that this list helps with discoverability.